clients/admin-ui/src/features/common/tablev2/PaginationBar.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  ChevronLeftIcon,
+  ChevronRightIcon,
+  HStack,
+  IconButton,
+  Text,
+} from "@fidesui/react";
+import { Table as TableInstance } from "@tanstack/react-table";
+
+type PaginationBarProps<T> = {
+  tableInstance: TableInstance<T>;
+};
+
+export const PaginationBar = <T,>({ tableInstance }: PaginationBarProps<T>) => {
```

Comment:
I didn't include the surrounding components above and below the table within the `FidesTable` component because it starts to complicate the logic and styling of it a lot.

It made things easier to grok by separating them like this.